lib/streamlit/elements/media.py

```diff
@@ -211,6 +213,9 @@ def video(
            https://doc-video-subtitles.streamlit.app/
            height: 700px
 
+        To see examples of all the supported subtitle input types, see
+        https://doc-video-subtitles2.streamlit.app/
```

Comment:
@snehankekre What app is this that you want to link to? If we're writing out a link to send people there externally, rather than an embedded app. We should choose a nicer URL.

Reply:
It's a MPA in wide mode and won't look great embedded. Switched to a 'nicer' url